forms-flow-api/Dockerfile

```diff
@@ -8,6 +8,7 @@ WORKDIR /forms-flow-api/app
 
 # install curl, gnupg2 and unzip
 RUN  apt-get update \
+  && apt-get dist-upgrade -y\
```

Comment:
Can you check if this code suggestion is valid 

<img width="1390" height="741" alt="Image" src="https://github.com/user-attachments/assets/b73aab0a-edf7-45b7-ae4c-cb767ffdb90d" />

Reply:
Updated